src/components/About/About.module.css

```diff
@@ -16,13 +22,57 @@
 }
 
 .repo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 500px;
   font-size: 25px;
   font-weight: 300;
-  margin: 0 auto;
+}
+
+.image {
+  width: 75%;
+  height: 75%;
+  border-radius: 50%;
 }
 
 .repolist {
-  font-size: 20px;
-  font-weight: 200;
+  list-style-type: none;
+  flex-direction: column;
+  border: 1px solid black;
+  border-radius: 5px;
   margin-bottom: 10px;
 }
+
+.link {
+  text-decoration: none;
+  color: black;
+}
+
+.link:hover {
+  background: grey;
+}
+
+.language {
+  position: relative;
+  min-width: 100px;
+  padding-left: 20px;
+}
+
+.python:before, .js:before {
+  content: "";
+  position: absolute;
+  border-radius: 50%;
+  left: 0;
+  bottom: 5px;
+  height: 12px;
+  width: 12px;
+}
+
+.python:before {
+  background: yellow;
+}
+
+.js:before {
+  background:red;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)